src/secrets/components/WarningPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {FC} from 'react'
+import {
+  AlignItems,
+  ComponentSize,
+  FlexDirection,
+  Gradients,
+  Icon,
+  IconFont,
+  JustifyContent,
+  Panel,
+} from '@influxdata/clockface'
+
+interface Props {
+  warningText: string
+}
+
+const WarningPanel: FC<Props> = ({warningText}) => (
+  <Panel
+    gradient={Gradients.LostGalaxy}
+    testID="panel"
+    border={true}
+    style={{marginBottom: '8px'}}
```

Comment:
let's try and use a clockface variable in a className-based CSS, for consistency

Reply:
Changed